public/js/app/components/statistics/__tests__/StatisticsDataTable.test.js

```diff
@@ -23,6 +23,7 @@ const statisticsResultForMemo = {
         courseCode: 'A31REA',
         courseMemoFileName: 'memo-123-3456.pdf',
         ladokRoundIds: ['1'],
```

Comment:
`ladokRoundIds` has to be removed to be sure tests working properly

Reply:
Removed